netmaster/objApi/apiController.go

```diff
@@ -1215,6 +1215,25 @@ func (ac *APIController) ServiceLBCreate(serviceCfg *contivModel.ServiceLB) erro
 
 	log.Infof("Received Service Load Balancer create: %+v", serviceCfg)
 
+	// Find the tenant
+	tenant := contivModel.FindTenant(serviceCfg.TenantName)
+	if tenant == nil {
+		return core.Errorf("Tenant:%s not found", serviceCfg.TenantName)
+	}
+
+	// Find the network
+	nwObjKey := serviceCfg.TenantName + ":" + serviceCfg.NetworkName
+	network := contivModel.FindNetwork(nwObjKey)
+	if network == nil {
+		return core.Errorf("Network %s not found", serviceCfg.NetworkName)
+	}
+
+	svcKey := serviceCfg.TenantName + ":" + serviceCfg.ServiceName
+	svc := contivModel.FindServiceLB(svcKey)
+	if svc == nil {
+		return core.Errorf("Service: %s not found", serviceCfg.ServiceName)
+	}
```

Comment:
the tenant check and network check is required because it was allowing me to create even an invalid tenant/network. so if teh tenant/network is not present, we need to throw an error.